Tests/DependencyInjection/Compiler/AddSwiftMailerTransportPassTest.php

```diff
@@ -26,19 +28,16 @@ class AddSwiftMailerTransportPassTest extends TestCase
 
     private $definition;
 
-    /**
-     * @before
-     */
-    protected function doSetUp()
+    protected function setUp(): void
     {
         $this->compilerPass = new AddSwiftMailerTransportPass();
-        $this->definition = $this->getMockBuilder('\Symfony\Component\DependencyInjection\Definition')->getMock();
+        $this->definition = $this->getMockBuilder(Definition::class)->getMock();
         $this->definition->expects($this->any())
             ->method('getArgument')
             ->with(0)
             ->willReturn(new Reference('swiftmailer'));
-        $this->container = $this->getMockBuilder('\Symfony\Component\DependencyInjection\ContainerBuilder')
-            ->setMethods(['getParameter', 'getDefinition', 'hasDefinition', 'addMethodCall'])->getMock();
+        $this->container = $this->getMockBuilder(ContainerBuilder::class)
```

Comment:
Do we really have to mock the `ContainerBuilder` class? What about using a "real" instance with "real" definitions instead?

Reply:
We probably don't. But I did not want to rewrite the whole test, tbh. 😓 

But I can look into it.

Reply:
we can do that first (see #496) and then you could rebase here